packages/hardhat-test-helpers/test/helpers/getStorageAt.ts

```diff
@@ -0,0 +1,41 @@
+import { assert } from "chai";
+import { BN } from "ethereumjs-util";
+import { ethers } from "ethers";
+
+import * as hh from "../../src";
+import { NumberLike } from "../../src/types";
+import { useEnvironment } from "../test-utils";
+
+describe("getStorageAt", function () {
+  useEnvironment("simple");
+  const account = "0x000000000000000000000000000000000000beef";
+  const code =
+    "0x000000000000000000000000000000000000000000000000000000000000beef";
+
+  it("should get the storage of a given address", async function () {
+    await hh.setStorageAt(account, "0x1", code);
+    await hh.mine();
```

Comment:
This `mine` is also unnecessary, unless I'm missing something. Same comment for the rest of the `hh.mine`s in the other tests.

Reply:
Whoops, you're right. That was just me mindlessly copy/paste/editing tests and not giving it as much thought as I should've 😅